96.md

```diff
@@ -164,7 +164,7 @@ The upload response is a json object as follows:
       // Could also be any url to download the file
       // (using or not using the /.well-known/nostr/nip96.json's "download_url" prefix),
       // for load balancing purposes for example.
-      ["url", "https://your-file-server.example/custom-api-path/719171db19525d9d08dd69cb716a18158a249b7b3b3ec4bbdec5698dca104b7b.png"],
+      ["url", "https://your-file-server.example/custom-api-path/pubkey/719171db19525d9d08dd69cb716a18158a249b7b3b3ec4bbdec5698dca104b7b.png"],
```

Comment:
Yes, especially when #1236 would allow the user to list their files and easily migrate them.

Reply:
The goal is to make it easy to know if a file was uploaded by a pubkey or another, it is clear that with a call to the API we can have it, but for a user it will not be easy.

I think this is in line with what blossom promotes, url's standards, in this case the original pubkey (in my opinion) is very relevant.

Reply:
The goal was to not rely on API calls to know if an image was uploaded by a pubkey. In case nobody sees the point, I give up 😂

 @arthurfranca @v0l @fishcakeday @sant0s12 
 
 I am going to close this PR, if you think there is something interesting in it you can add it to the other PR's that are open.

Reply:
The spirit of this PR will live on `https://link.to/<image-sha256>.png#ox=<hash>&nip96u=<pubkey>` kind of urls if nip96u tag is added heheh